watertap/costing/units/electrodialysis.py

```diff
@@ -13,41 +13,23 @@
 import pyomo.environ as pyo
 from ..util import (
     register_costing_parameter_block,
+    cost_rectifier,
     make_capital_cost_var,
     make_fixed_operating_cost_var,
 )
 
 
 def build_electrodialysis_cost_param_block(blk):
 
-    blk.cem_membrane_cost = pyo.Var(
-        initialize=43,
-        doc="Cost of CEM membrane used in Electrodialysis ($/CEM/area)",
+    blk.membrane_capital_cost = pyo.Var(
+        initialize=150,
+        doc="Membrane and capitcal costs in [US$/m^2-membrane-area], referenced to Desalination 142 (2002) 267-286.",
```

Comment:
```suggestion
        doc="Stack capital cost in [US$/m^2-membrane area], which includes costs for membranes and other components within the stack",
```
I might also mention the reference (`"referenced to Desalination 142 (2002) 267-286."`) outside of this doc string